src/main/java/com/bootcamp/demo/repository/FirebaseSessionRepository.java

```diff
@@ -0,0 +1,35 @@
+package com.bootcamp.demo.repository;
+
+import com.bootcamp.demo.model.Session;
+import com.google.cloud.firestore.Firestore;
+
+
+import java.util.Set;
+
+public class FirebaseSessionRepository implements SessionRepository{
+    Firestore firestoreDB;
+
+    public FirebaseSessionRepository(Firestore firestoreDB) {
+        this.firestoreDB = firestoreDB;
+    }
+
+    @Override
+    public Set<Session> findAll() {
+        return null;
+    }
+
+    @Override
+    public String save(Session session) {
+        return null;
+    }
+
+    @Override
+    public void remove(Session o) {
+
+    }
+
+    @Override
+    public Session findById(String id) {
+        return null;
+    }
+}
```

Comment:
Avem nevoie si de un update(Session sessions) - il apelam cand incheiem o sesiune si ii modificam endSession

Reply:
save-ul face update cand gaseste id-ul in db

Reply:
Am inteles, perfect atunci.